lib/libutee/tee_api_operations.c

```diff
@@ -763,39 +763,17 @@ static TEE_Result tee_buffer_update(
 
 	if (slen >= (buffer_size + buffer_left)) {
 		/* Buffer is empty, feed as much as possible from src */
-		if (TEE_ALIGNMENT_IS_OK(src, uint32_t)) {
```

Comment:
Right. Accelerated algorithms may be impacted.
Let's wait for @jforissier before making a status on it.